ergo-wallet/src/main/scala/org/ergoplatform/contracts/ReemissionContracts.scala

```diff
@@ -0,0 +1,110 @@
+package org.ergoplatform.contracts
+
+import org.ergoplatform.ErgoBox.R2
+import org.ergoplatform.ErgoScriptPredef.{boxCreationHeight, expectedMinerOutScriptBytesVal}
+import org.ergoplatform.{Height, MinerPubkey, Outputs, Self}
+import org.ergoplatform.settings.MonetarySettings
+import sigmastate.{AND, EQ, GE, GT, LE, Minus, OR, SByte, SCollection, SLong, STuple}
+import sigmastate.Values.{ByteArrayConstant, ErgoTree, IntConstant, LongConstant}
+import sigmastate.utxo.{ByIndex, ExtractAmount, ExtractRegisterAs, ExtractScriptBytes, OptionGet, SelectField, SizeOf}
+
+/**
+  * Container for re-emission related contracts. Contains re-emission contract and pay-to-reemission contract.
+  */
+trait ReemissionContracts {
+
+  /**
+    * How much miner can take per block from re-emission contract
+    */
+  val reemissionRewardPerBlock: Long = 3 * 1000000000L // 3 ERG
```

Comment:
there is this `EmissionRules.CoinsInOneErgo` constant for this

Reply:
Re-used it!